deltacat/compute/compactor/compaction_session.py

```diff
@@ -445,6 +445,25 @@ def _execute_compaction_round(
     )
     logger.info(f"Getting {len(mat_tasks_pending)} materialize result(s)...")
     mat_results = ray.get(mat_tasks_pending)
+    total_count_of_src_dfl_not_touched_list = [
+        m.count_of_src_dfl_not_touched for m in mat_results
+    ]
+    total_length_src_dfl_list = [m.count_of_src_dfl for m in mat_results]
+    total_count_of_src_dfl_not_touched = sum(total_count_of_src_dfl_not_touched_list)
+    total_length_src_dfl = sum(total_length_src_dfl_list)
+    logger.info(
+        f"Got total of {total_count_of_src_dfl_not_touched} manifest files not touched."
+    )
+    logger.info(
+        f"Got total of {total_length_src_dfl} manifest files during compaction."
+    )
+    untouched_manifest_file_percentage = (
+        round(total_count_of_src_dfl_not_touched / total_length_src_dfl, 3) * 100
+    )
+    logger.info(
+        f"{untouched_manifest_file_percentage} percent of manifest files are not touched during materialize."
```

Comment:
Can we persist the skipping file ratio to the round completion file? We thought about using ddb, but it seems unless we return through compaction_session to compaction driver, or we figure out how to retrieve from cloudwatch in compaction driver, the easiest way might be through the rcf.